src/main/kotlin/blackjack/view/OutputView.kt

```diff
@@ -0,0 +1,42 @@
+package blackjack.view
+
+import blackjack.constant.Messages
+import blackjack.domain.User
+import blackjack.domain.Users
+
+/**
+ * 출력을 담당하는 클래스
+ * Created by Jaesungchi on 2022.06.07..
+ */
+object OutputView {
+    fun printHandOutMessage(users: Users) {
+        println()
+        println(Messages.HAND_OUT_CARD.format(users.users.joinToString(", ") { it.name }))
```

Comment:
캡슐화하고 일급컬렉션에 책임을 추가해 볼까요?
```suggestion
        println(Messages.HAND_OUT_CARD.format(users.names().joinToString()))
```